### PR TITLE
[PDI-18308] - new connection type for Amazon Athena

### PR DESCRIPTION
**Pull request by abryant-hv**

### CODE REVIEW

File: dbdialog/src/main/java/org/pentaho/ui/database/event/athena/AthenaHandler.java [hunk 1]
```diff
@@ -0,0 +1,204 @@
+package org.pentaho.ui.database.event.athena;
```
Comment (peterrinehart) [inline dbdialog/src/main/java/org/pentaho/ui/database/event/athena/AthenaHandler.java:1]: Needs a copyright header.

**Review by peterrinehart — CHANGES_REQUESTED** [review]

**Comment by buildguy** [comment]
### :x: Build failed in 44m 39s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl core,dbdialog
```
---
:ok_hand: All tests passed!
 
Tests run: **5284**, Failures: **0**, Skipped: **11** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-9208/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by peterrinehart — APPROVED** [review]

**Comment by buildguy** [comment]
### :x: Build failed in 45m 2s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl core,dbdialog
```
---
:ok_hand: All tests passed!
 
Tests run: **5348**, Failures: **0**, Skipped: **14** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-9208/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by tgf — APPROVED** [review]

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :warning: Build finished in 41m 42s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox -pl core,dbdialog
```
---
### :no_entry: Failed Tests

#### :cloud_with_lightning_and_rain: 1 test(s) failed:

<details><summary><b>org.pentaho.di.trans.steps.excelwriter.ExcelWriterStepMetaTest.testRoundTrip</b> (click to expand)</summary><p>

```
Error validating create_parent
```

</p></details>

  
Tests run: **5348**, Failures: **1**, Skipped: **14** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-9208/4/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 43m 41s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox -pl core,dbdialog
```
---
:ok_hand: All tests passed!
 
Tests run: **5373**, Failures: **0**, Skipped: **14** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/pdi/job/PR-9208/5/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**